doc/interfaces/dg.rst

```diff
@@ -0,0 +1,43 @@
+DG Interface
+================
+
+Interface to `DG Technologies <http://dgtech.com>`__ Beacon
+
+
+Bus
+---
+
+.. autoclass:: can.interfaces.dg.dg.dgBus
+
+
+Configuration
+-------------
+There are 6 parameters accepted by the ``dgBus`` class::
```

Comment:
Document using Sphinx instead.

Reply:
When I build the docs using Sphinx from the directions in the developers tutorial, I just get html files in doc/_build. Is there any specific style I should be adhering to when doing the rst documentation?

Reply:
Sorry, that was confusing of me. You can have a look at some other interface init method’s docstring for examples. You could also utilize the typing module to document the types accepted by the parameters but that’s not required (I think).

Reply:
Done.